app-pda/pilot-link/files/pilot-link-0.12.5-C23.patch

```diff
@@ -0,0 +1,85 @@
+Correct build for C23
+Wrong type for size variable, missing parameters in decls
+Missing include and wrong types in bundled libusb library
+https://bugs.gentoo.org/944433
+https://bugs.gentoo.org/883101
+--- a/src/parsedate.c
++++ b/src/parsedate.c
+@@ -175,8 +175,8 @@
+ static time_t	yyRelSeconds;
+ 
+ 
+-extern struct tm	*localtime();
+-static void		date_error();
++extern struct tm	*localtime(const time_t *timep);
++static void		date_error(char *s);
+ 
+ 
+ /* Enabling traces.  */
+--- a/src/parsedate.y
```

Comment:
Modifying a .y means you need to check for lex/yacc deps.

Reply:
In BDEPEND? Will do. I think it already uses bison but without noting it in ebuild.

Reply:
yes